stdlib/public/core/CollectionDifference.swift

```diff
@@ -222,7 +222,11 @@ public struct CollectionDifference<ChangeElement> {
 ///   }
 /// }
 /// ```
+<<<<<<< HEAD
```

Comment:
Probably wasn't intended to be committed. There are a few of them.

Reply:
Yeah, I’m reworking some of this anyway to minimize the churn.